falcon/response.py

```diff
@@ -554,6 +554,22 @@ def add_link(self, target, rel, title=None, title_star=None,
         """,
         lambda v: ', '.join(v))
 
+    accept_ranges = header_property(
+        'Accept-Ranges',
+        """
+        Sets the Accept-Ranges headers.
+
+        The "Accept-Ranges" header field allows a server to indicate that it
+        supports range requests for the target resource.
```

Comment:
That works. Just a couple of nitpicks though: remove the quotes around the header name, but do quote the "bytes" range unit.